bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/obsrecord/PrivateVisit.java

```diff
@@ -88,191 +88,8 @@ Site divineSite() {
-        long nightTime;
-
-        long getDaytime() {
-            return totalTime - nightTime;
-        }
-    }
-
-    static void computeEventInterval(EventData e1, EventData e2, EventInterval inv) {
-        long t1 = e1.evt.getTimestamp();
-        long t2 = e2.evt.getTimestamp();
-
-        // assertion
-        if (t1 > t2) {
-            throw new IllegalArgumentException("out of order events");
-        }
-
-        inv.totalTime = t2 - t1;
-        inv.nightTime = 0;
-
-        if (e1.obsNight == null) {
-            // no site information, so just assume it is all night time
-            // this shouldn't happen in practice
-            inv.nightTime = inv.totalTime;
-            return;
-        }
-
-        long n1start = e1.darkNight.getStartTime();
-        long n1end   = e1.darkNight.getEndTime();
-
-        long start = t1;  // assume t1 is in the dark part of the obs night 1
-        if (t1 < n1start) {
-            // was before the dark time begins on obs night 1
-            start = n1start;
-        } else if (t1 >= n1end) {
-            // was after the dark time is over on obs night 1
-            start = -1;
-        }
-
-        long n2start = e2.darkNight.getStartTime();
-        long n2end   = e2.darkNight.getEndTime();
-
-        long end = t2; // assume t2 is in the dark park of the obs night 2
-        if (t2 < n2start) {
-            // was before the dark time begins on obs night 2
-            end = -1;
-        } else if (t2 >= n2end) {
-            end = n2end;
-        }
-
-        // First handle the common case, both events are in the same observing
-        // night.
-        if (e1.obsNight.includes(t2)) {
-            if ((start != -1) && (end != -1)) {
-                inv.nightTime = end - start;
-            } // else no nighttime for this event
-            return;
-        }
-
-        // Now handle the unusual case, events on different observing nights.
-        long nighttime = 0;
-
-        // First get the amount of nighttime on the first night.
-        if (start != -1) {
-            nighttime = n1end - start;
-        }
-
-        // Add in the amount of nighttime on the last night
-        if (end != -1) {
-            nighttime += end - n2start;
-        }
-
-        // Now, if there are observing nights between the two events, add in
-        // all the night time for those nights.
-        Site site = e1.obsNight.getSite();
-        ObservingNight curNight = new ObservingNight(site, e1.obsNight.getEndTime());
-        while (curNight.getStartTime() < e2.obsNight.getStartTime()) {
-            Night darkNight = TwilightBoundedNight.forObservingNight(TwilightBoundType.NAUTICAL, curNight);
-            nighttime += darkNight.getTotalTime();
-            curNight   = new ObservingNight(site, curNight.getEndTime());
-        }
-        inv.nightTime = nighttime;
-    }
-
-    VisitTimes getTimeCharges(ObsQaRecord qa,
-                              ConfigStore store) {
```

Comment:
All of this is just the original version that was moved into `PrimordialVisitCalculator`.